h2/src/main/org/h2/table/InformationSchemaTable.java

```diff
@@ -2847,11 +2847,16 @@ private void sessions(SessionLocal session, ArrayList<Row> rows, SessionLocal s)
         NetworkConnectionInfo networkConnectionInfo = s.getNetworkConnectionInfo();
         Command command = s.getCurrentCommand();
         int blockingSessionId = s.getBlockingSessionId();
+        //Fix bug the NullPointerException caused by accessing 'user' after the session has been closed.
+        User user = s.getUser();
+        if(user==null) {
+            return;
+        }
```

Comment:
Usually we don't add comments about fixed bugs to the source code.

You can add a comment like this instead:
```Java
if (user == null) {
    // Session was closed concurrently
    return;
}
```
Please also move this code to the start of this method and use the same code style as the whole project (`if (user == null)` instead of `if(user=null)`).

Reply:
Update code with the same code style